src/pipeline_blocks.py

```diff
@@ -225,49 +223,53 @@ def classifier_xgb(features, config, train_mode, suffix, **kwargs):
     return xgboost
 
 
-def classifier_sklearn(sklearn_features,
+def classifier_sklearn(features,
                        ClassifierClass,
-                       full_config,
+                       config,
                        clf_name,
                        train_mode,
                        suffix,
-                       normalize,
                        **kwargs):
-    config, model_params, rs_config = full_config
+    model_name = '{}{}'.format(clf_name, suffix)
+    model_params = getattr(config, clf_name)
+    random_search_params = getattr(config.random_search, clf_name)
     if train_mode:
-        if config.random_search.random_forest.n_runs:
-            transformer = RandomSearchOptimizer(
-                partial(get_sklearn_classifier,
-                        ClassifierClass=ClassifierClass,
-                        normalize=normalize),
-                model_params,
-                train_input_keys=[],
-                valid_input_keys=['X_valid', 'y_valid'],
-                score_func=roc_auc_score,
-                maximize=True,
-                n_runs=rs_config.n_runs,
-                callbacks=[NeptuneMonitor(**rs_config.callbacks.neptune_monitor),
-                           PersistResults(**rs_config.callbacks.persist_results)]
-            )
+        features_train, features_valid = features
+        if getattr(config.random_search, clf_name).n_runs:
```

Comment:
@kstrzala since we have already fetched the `random_search_params` lets just use `random_search_params.n_runs`